src/hotspot/share/gc/serial/serialHeap.hpp

```diff
@@ -252,7 +252,7 @@ class SerialHeap : public CollectedHeap {
   };
 
  protected:
-  virtual void gc_prologue(bool full);
+  virtual void gc_prologue();
```

Comment:
The `gc_prologue` and `gc_epilogue` don't need to be marked as `virtual`. This can be done at another ticket.